modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -905,32 +906,45 @@ BOOST_SERIALIZATION_SPLIT_FREE(DomainInfo);
 BOOST_IS_BITWISE_SERIALIZABLE(ComboAddress);
 
 template <>
-std::string serToString(const LMDBBackend::LMDBResourceRecord& lrr)
```

Comment:
I'm not convinced `value` is a better name than `lrr` here, but I don't feel strongly about it

Reply:
Same here.